static/js/services/db.js

```diff
@@ -134,8 +134,7 @@ var utils = require('kujua-utils'),
               .changes({
                 live: true,
                 since: 'now',
-                doc_ids: [ '_design/medic' ],
-                timeout: 1000 * 60 * 60
```

Comment:
Yeah. It doesn't do what we thought it did. If you specific a heartbeat (which pouch does by default) then that keeps refreshing the timeout anyway, so this timeout will only fire if the connection is dropped and not cleaned up. If this happens we should cancel the request much sooner, so the default is appropriate.